gpflow/optimizers/natgrad.py

```diff
@@ -79,7 +79,12 @@ def __init__(self, gamma: Scalar, name=None):
         super().__init__(name)
         self.gamma = gamma
 
-    def minimize(self, loss_fn: Callable, var_list: List[Parameter]):
+    def minimize(
+            self,
+            loss_fn: Callable,
+            var_list: List[
+                Union[Tuple[Parameter, Parameter], Tuple[Parameter, Parameter, XiTransform]]]
```

Comment:
:+1:

Reply:
i did notice in this that this type signature is incompatible with that of `tf.optimizers.Optimizer`. we can come back to that